std/meta.d

```diff
@@ -1120,42 +1120,23 @@ template templateOr(Preds...)
     }
 }
 
+import std.traits : isIterable;
```

Comment:
Imports go at the top of the file.

Reply:
The only existing top-level import in the file was not at the top, so I tried to stay in keeping. I will move both.